test/pycheck/explain_test.py

```diff
@@ -2,7 +2,7 @@
 
 
 def test_explain(cur: Cursor):
-    cur.sql("CREATE TABLE test_table (id int primary key, name text)")
+    cur.sql("CREATE TABLE test_table (id int, name text)")
```

Comment:
I needed to change this definition to not create an index because index scans are currently broken, see #183 for details.